example/src/App.styles.ts

```diff
@@ -1,4 +1,3 @@
-import { Colors } from "react-native/Libraries/NewAppScreen";
```

Comment:
The type of `Colors` was resolved as `any`, so now `@typescript-eslint/no-explicit-any` rule complains